src/panels/lovelace/editor/card-editor/show-select-view-dialog.ts

```diff
@@ -0,0 +1,34 @@
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { Lovelace } from "../../types";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "show-select-view": SelectViewDialogParams;
+  }
+}
+
+let registeredDialog = false;
```

Comment:
We don't do this anymore. You're copy pasting from the wrong file 😆 

Try this https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/dialogs/zha-device-info-dialog/show-dialog-zha-device-info.ts#L1